spec/requests/start_spec.rb

```diff
@@ -11,12 +11,13 @@
 
   context 'nearing end of cycle' do
     before do
-      allow(Settings).to receive(:cycle_ending_soon).and_return(true)
+      activate_feature(:cycle_ending_soon)
+      deactivate_feature(:cycle_has_ended)
       Rails.application.reload_routes!
     end
 
     after do
-      allow(Settings).to receive(:cycle_ending_soon).and_return(false)
+      deactivate_feature(:cycle_ending_soon)
```

Comment:
Why is this needed? I thought that the stubs would automatically get cancelled but the tests fail without these 'resets'

Reply:
I assumed this was for the route reloading, required because of the conditionals present here https://github.com/DFE-Digital/find-teacher-training/blob/master/config/routes.rb#L11-L33

Reply:
ah yes that makes sense, thanks!